### PR TITLE
update French translations

### PR DESCRIPTION
**Pull request by sebdalgarno**
see #175 , #174 
also update multi_ci arg which was now failing due to stricter ...

### CODE REVIEW

**Comment by sebdalgarno** [comment]
waiting on joes confirmation regarding `ci_method`

**Comment by joethorley** [comment]
Yes I can confirm that we should just go with the default `ci_method` (which is `"weighted_samples"`) but also you can drop `multi_est = TRUE` as this is  the new default.

**Comment by sebdalgarno** [comment]
deployed to main poisson site -
will merge once BCANZ translations added - then merge into bcgov fork

File: data-raw/data.R
Comment (joethorley) [inline data-raw/data.R:88]:
```suggestion
    french = "Étiquette de changement"
```

Do we need this capitalized?

File: inst/extdata/user-fr.md
Comment (joethorley) [inline inst/extdata/user-fr.md:1]:
```suggestion
Cette application web estime la distribution de la sensibilité des espèces en fonction des données de concentration. L’application est conçue à partir ddu progiciel R [ssdtools](https://github.com/bcgov/ssdtools), et partage les mêmes fonctions.
De plus amples informations sur les méthodes de ssdtools sont disponibles aux liens suivants [articles du site web] (https://bcgov.github.io/ssdtools/articles/).
```

Comment (joethorley) [reply inst/extdata/user-fr.md:1]: Can you put new sentences on new lines here and other places.

**Review by joethorley — CHANGES_REQUESTED** [review]

**Review by joethorley — CHANGES_REQUESTED** [review]
Looks great - I've added a few suggested edits for you to review.

File: data-raw/data.R
Comment (sebdalgarno) [reply data-raw/data.R:88]: yes good catch

**Review by joethorley — APPROVED** [review]
looks great

**Comment by sebdalgarno** [comment]
deployed to poisson site and tested

**Pull request merged**
